pmm-app/src/pmm-dbaas/DBaaS.styles.ts

```diff
@@ -1,8 +1,12 @@
 import { CSSProperties } from 'react';
+import { css } from 'emotion';
 
 export const styles = {
   disabled: {
     opacity: 0.6,
     pointerEvents: 'none',
   } as CSSProperties,
+  panelContentWrapper: css`
+    margin-top: 20px;
```

Comment:
Change this object to a getStyles function and use theme values.

Reply:
https://jira.percona.com/browse/PMM-7027 yes, we already have a ticket for it so it will be done there